docs/framework/wcf/feature-details/how-to-create-temporary-certificates-for-use-during-development.md

```diff
@@ -7,50 +7,44 @@ helpviewer_keywords:
+>  The certificates New-SelfSignedCertificate cmdlet generates are provided for testing purposes only. When deploying a service or client, be sure to use an appropriate certificate provided by a certification authority. This could either be from a [!INCLUDE[ws2003](../../../../includes/ws2003-md.md)] certificate server in your organization or a third party.  
 >   
->  By default, the [Makecert.exe (Certificate Creation Tool)](http://msdn.microsoft.com/library/b0343f8e-9c41-4852-a85c-f8a0c408cf0d) creates certificates whose root authority is called "Root Agency**."** Because the "Root Agency" is not in the Trusted Root Certification Authorities store, this makes these certificates insecure. Creating a self-signed certificate that is placed in the Trusted Root Certification Authorities store enables you to create a development environment that more closely simulates your deployment environment.  
+>  By default, the [New-SelfSignedCertificate](https://docs.microsoft.com/en-us/powershell/module/pkiclient/new-selfsignedcertificate?view=win10-ps) creates certificates whose root authority is called "Root Agency**."** Because the "Root Agency" is not in the Trusted Root Certification Authorities store, this makes these certificates insecure. Creating a self-signed certificate that is placed in the Trusted Root Certification Authorities store enables you to create a development environment that more closely simulates your deployment environment.  
   
  For more information about creating and using certificates, see [Working with Certificates](../../../../docs/framework/wcf/feature-details/working-with-certificates.md). For more information about using a certificate as a credential, see [Securing Services and Clients](../../../../docs/framework/wcf/feature-details/securing-services-and-clients.md). For a tutorial about using Microsoft Authenticode technology, see [Authenticode Overviews and Tutorials](http://go.microsoft.com/fwlink/?LinkId=88919).  
   
 ### To create a self-signed root authority certificate and export the private key  
   
-1.  Use the MakeCert.exe tool with the following switches:  
-  
-    1.  `-n` `subjectName`. Specifies the subject name. The convention is to prefix the subject name with "CN = " for "Common Name".  
-  
-    2.  `-r`. Specifies that the certificate will be self-signed.  
-  
-    3.  `-sv` `privateKeyFile`. Specifies the file that contains the private key container.  
-  
-     For example, the following command creates a self-signed certificate with a subject name of "CN=TempCA."  
-  
-    ```  
-    makecert -n "CN=TempCA" -r -sv TempCA.pvk TempCA.cer  
-    ```  
-  
-     You will be prompted to provide a password to protect the private key. This password is required when creating a certificate signed by this root certificate.  
-  
+The following command creates a self-signed certificate with a subject name of "RootCA" in the user MY store. 
```

Comment:
> MY  [](start = 99, length = 4)

It's only called My in .Net (so Powershell by extension). It's the user Personal store in the certificate manager so it should be referred to as that in documentation.